### PR TITLE
make mellanox_fw_update script to be compatible with python3

### PR DESCRIPTION
**Pull request by wmousa**
make mellanox_fw_update script to be compatible with python3

### CODE REVIEW

File: templates/firstboot/mellanox_fw_update.yaml [hunk 1]
```diff
@@ -27,11 +27,11 @@ parameters:
     description: 'Link type ["eth", "ib"]'
   ESWITCH_IPV4_TTL_MODIFY_ENABLE:
     type: boolean
-    default: True
+    default: False
```
Comment (adrianchiris) [inline templates/firstboot/mellanox_fw_update.yaml:30]: why was this changed?

File: templates/firstboot/mellanox_fw_update.yaml [hunk 2]
```diff
@@ -27,11 +27,11 @@ parameters:
...
     description: 'Enable TTL modification by E-Switch'
   PRIO_TAG_REQUIRED_EN:
     type: boolean
-    default: True
+    default: False
```
Comment (adrianchiris) [inline templates/firstboot/mellanox_fw_update.yaml:34]: why was this changed?

File: templates/firstboot/mellanox_fw_update.yaml (hunk 1 above)
Comment (wmousa) [reply templates/firstboot/mellanox_fw_update.yaml:30]: So the default should be False, but we didn't rebase the changes on my local repo to this Mellanox repo

File: templates/firstboot/mellanox_fw_update.yaml (hunk 2 above)
Comment (wmousa) [reply templates/firstboot/mellanox_fw_update.yaml:34]: same as above

File: templates/firstboot/mellanox_fw_update.yaml [hunk 3]
```diff
@@ -49,14 +49,16 @@ resources:
...
-            from HTMLParser import HTMLParser
             import logging
             import os
             import re
             import shutil
             import tempfile
             import threading
-            import urllib2
+
+            from six.moves.html_parser import HTMLParser
```
Comment (moshe010) [inline templates/firstboot/mellanox_fw_update.yaml]: I would prefer that you import the module  html_parser and not the class HTMLParser.
I see that it was originally written like this, but it inconsistent other imports

Comment (wmousa) [reply templates/firstboot/mellanox_fw_update.yaml]: done

**Review by moshe010 — APPROVED** [review]

**Review by moshe010 — APPROVED** [review]
